contracts/proxy/Clones.sol

```diff
@@ -229,7 +229,7 @@ library Clones {
     function fetchCloneArgs(address instance) internal view returns (bytes memory) {
         bytes memory result = new bytes(instance.code.length - 0x2d); // revert if length is too short
         assembly ("memory-safe") {
-            extcodecopy(instance, add(result, 0x20), 0x2d, mload(result))
```

Comment:
Any rational to changing that 0x20 in 32 but keeping `0x2d` as an hex?

If I had the change something, that would be the second one. There are many places where we do `add(..., 0x20)`: ERC2771Forwarder, Bytes, Create2, SlotDerivation, Arrays, Base64, Checkpoints, Math, RSA, P256, ECDSA, ERC1155, SAfeERC20 ...

There are way fewer places where we do `add(..., 32)`